### ISSUE TITLE
NTR: skin emphysema

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#10623]
**Preferred term label:**
skin emphysema

**Synonyms**
Subcutaneous emphysema (SCE)

**Definition (free text, please give PubMed ID)**
Skin emphysema/Subcutaneous emphysema occurs when gas or air accumulates and seeps under the skin, where normally no gas should be present. While the presence of air in subcutaneous or mediastinal tissue is not dangerous in itself, prompt recognition of the underlying cause is essential. Subcutaneous emphysema can be caused by medical procedures and medical conditions that cause the pressure in the alveoli of the lung to be higher than that in the tissues outside of them. Its most common causes are pneumothorax or a chest tube that has become occluded by a blood clot or fibrinous material. It can also occur spontaneously due to rupture of the alveoli, with dramatic presentation. When the condition is caused by surgery it is called surgical emphysema. The term spontaneous subcutaneous emphysema is used when the cause is not clear.

**Parent term (use hpo.jax.org/app)**
Abnormal epidermal morphology HP:0011124

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-0113-912X, 0000-0002-7371-8158

**Comment by pnrobinson** [comment]
@GrieseAG 
Adding new term: 
- ID: HP:6001021
- label: Subcutaneous emphysema
- parents: [Abnormal epidermal morphology (HP:0011124)](https://hpo.jax.org/app/browse/term/HP:0011124)
- pmid: PMID:31194349
- def: Skin emphysema/Subcutaneous emphysema occurs when gas or air accumulates and seeps under the skin, where normally no gas should be present. While the presence of air in subcutaneous or mediastinal tissue is not dangerous in itself, prompt recognition of the underlying cause is essential.
- comment: Subcutaneous emphysema can be caused by medical procedures and medical conditions that cause the pressure in the alveoli of the lung to be higher than that in the tissues outside of them. Its most common causes are pneumothorax or a chest tube that has become occluded by a blood clot or fibrinous material. It can also occur spontaneously due to rupture of the alveoli, with dramatic presentation. When the condition is caused by surgery it is called surgical emphysema. The term spontaneous subcutaneous emphysema is used when the cause is not clear.
- orcid: orcid:0000-0002-7371-8158
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10623
- synonyms: Skin emphysema(NONE)
